tests/query.json

```diff
@@ -200,7 +209,13 @@
         ["QueryCount", 3, "StringObject", "stringCol ==[c] 'a'"],
         ["QueryCount", 5, "StringObject", "stringCol BEGINSWITH[c] 'A'"],
         ["QueryCount", 4, "StringObject", "stringCol ENDSWITH[c] 'c'"],
-        ["QueryCount", 2, "StringObject", "stringCol CONTAINS[c] 'B'"]
+        ["QueryCount", 2, "StringObject", "stringCol CONTAINS[c] 'B'"],
+        ["QueryCount", 2, "StringObject", "stringCol LIKE[c] '*B?'"],
+        ["QueryCount", 0, "StringObject", "stringCol LIKE 1"],
+        ["QueryCount", 0, "StringObject", "stringCol LIKE LIKE"],
+        ["QueryCount", 0, "StringObject", "stringCol LIKE[A] '*BC'"],
```

Comment:
(Sorry, I didn't answer your question. It's an invalid flag and the query should return 0 results.)